.github/workflows/tests.yml

```diff
@@ -69,22 +69,21 @@ jobs:
   linux-tests:
     name: Linux Test
     runs-on: ubuntu-22.04
-    strategy:
-      matrix:
-        target: [
-          x86_64-unknown-linux-musl,
-          i686-unknown-linux-gnu,
-          i686-unknown-linux-musl,
-        ]
     steps:
       - uses: actions/checkout@v3
       - uses: dtolnay/rust-toolchain@stable
         with:
-          targets: ${{ matrix.target }}
-      - name: Install multilib
-        run: sudo apt-get update && sudo apt-get install gcc-multilib
+          targets: x86_64-unknown-linux-musl, i686-unknown-linux-gnu, i686-unknown-linux-musl, x86_64-unknown-linux-gnux32
+      - name: Install libc and libgcc for 32-bit targets
+        run: |
+          sudo apt-get update
+          sudo apt-get install --no-install-recommends libc6-dev-i386 libc6-dev-x32 lib32gcc-11-dev libx32gcc-11-dev
       - uses: Swatinem/rust-cache@v2
-      - run: cargo test --target=${{ matrix.target }} --features=std
+      - run: cargo test --target=x86_64-unknown-linux-musl --features=std
+      - run: cargo test --target=i686-unknown-linux-gnu --features=std
+      - run: cargo test --target=i686-unknown-linux-musl --features=std
+      # TODO: Find a Linux image/runner with CONFIG_X86_X32_ABI set
+      - run: cargo test --no-run --target=x86_64-unknown-linux-gnux32 --features=std
```

Comment:
We should also include the rdrand feature for the x86_64 targets, especially x32.

Reply:
Note that the `rdrand` feature doesn't do anything on supported targets. However, we do test the rdrand implementation on all x86/x86_64 targets (see `tests/rdrand.rs`). See how [this run](https://github.com/rust-random/getrandom/actions/runs/9423752238/job/25962770313) runs `tests/rdrand.rs`.